testenv_arm_joint_controller:
  type: "effort_controllers/JointTrajectoryController"
  joints:
    - testenv_joint_1
    - testenv_joint_2
    - testenv_joint_3
    - testenv_joint_4
    - testenv_joint_5
    - testenv_joint_6

  gains: # Required because we're controlling an effort interface
    testenv_joint_1: {p: 120000,  d: 22000, i: 46000, i_clamp: 1} 
    testenv_joint_2: {p: 120000,  d: 22000, i: 46000, i_clamp: 1} 
    testenv_joint_3: {p: 120000,  d: 22000, i: 46000, i_clamp: 1}
    testenv_joint_4: {p: 2000,  d: 120, i: 40, i_clamp: 1} 
    testenv_joint_5: {p: 3000,  d: 140, i: 20, i_clamp: 1}
    testenv_joint_6: {p: 12000,  d: 80, i: 20, i_clamp: 1}

